Cabal/Distribution/Simple/GHC.hs

```diff
@@ -127,7 +126,7 @@ configure verbosity hcPath hcPkgPath conf0 = do
 
   (ghcProg, ghcVersion, progdb1) <-
     requireProgramVersion verbosity ghcProgram
-      (orLaterVersion (mkVersion [6,11]))
+      (orLaterVersion (mkVersion [7,6]))
```

Comment:
Revert.

Reply:
I bumped it up to 7.0.1 (we test on Travis with 7.0.4, I have discovered). Doing that lets us simplify `PathsModule.hs` above. I can revert these if we really want to avoid changing this, but there should be no-one running an ancient development snapshot (right?).

Reply:
ping @hvr.

Reply:
Github keeps hiding this section after I push. Let's see if replying will keep it open...